### PR TITLE
[GEOT-5641] Mollweide projection misses parameter

### PR DESCRIPTION
**Pull request by marpet**
[GEOT-5641] Mollweide projection misses parameter

### CODE REVIEW

File: modules/library/referencing/src/main/java/org/geotools/referencing/operation/projection/Mollweide.java [hunk 1]
```diff
@@ -34,12 +34,12 @@
 
 /**
  * Mollweide projection
- * 
+ *
```
Comment (aaime) [inline modules/library/referencing/src/main/java/org/geotools/referencing/operation/projection/Mollweide.java]: Full of whitespace changes, please remove

File: modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/MollweideProviderTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import org.opengis.parameter.ParameterDescriptorGroup;
+import org.opengis.parameter.ParameterValueGroup;
+import org.opengis.referencing.operation.MathTransform;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Marco Peters
+ */
+public class MollweideProviderTest {
```
Comment (aaime) [inline modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/MollweideProviderTest.java]: Could you also add a script test checking the reprojection values against a reference generated by cs2cs? Check the existing Mollweide.txt and extend it.

**Comment by marpet** [comment]
Done. White spaces are removed and the script is extended.

File: modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/MollweideProviderTest.java (hunk 2 above)
Comment (marpet) [inline modules/library/referencing/src/test/java/org/geotools/referencing/operation/projection/MollweideProviderTest.java]: The script is updated.

**Pull request merged**